renovate.json

```diff
@@ -682,16 +684,884 @@
       ],
       "labels": [
         "Team:Operations",
+        "release_note:skip",
+        "backport:all-open"
+      ],
+      "minimumReleaseAge": "7 days",
+      "enabled": true
+    },
+    {
+      "groupName": "backport",
+      "matchDepNames": [
+        "backport"
+      ],
+      "matchDepPatterns": [],
+      "reviewers": [
+        "team:kibana-operations"
+      ],
+      "matchBaseBranches": [
+        "main"
+      ],
+      "labels": [
+        "Team:Operations",
+        "release_note:skip",
+        "backport:all-open"
+      ],
+      "minimumReleaseAge": "7 days",
+      "enabled": true
+    },
+    {
+      "groupName": "bazel",
+      "matchDepNames": [],
+      "matchDepPatterns": [
+        "^@bazel"
+      ],
+      "reviewers": [
+        "team:kibana-operations"
+      ],
+      "matchBaseBranches": [
+        "main"
+      ],
+      "labels": [
+        "Team:Operations",
+        "release_note:skip",
+        "backport:all-open"
+      ],
+      "minimumReleaseAge": "7 days",
+      "enabled": true
+    },
+    {
+      "groupName": "typescript",
+      "matchDepNames": [
+        "typescript",
+        "@types/node"
```

Comment:
`typescript` is almost always going to require a manual update, planning and coordination with other teams. So, we have opted not to have it in `renovate` for now. Related [#189415](https://github.com/elastic/kibana/issues/189415)

`@types/node` will be upgrade with Node version bumps in the future. Planning to add that to `renovate` in the near future.

Reply:
Same as above, I'll mark this as `enabled: false` for the time being. I'll remove `@types/node`